pkgs/development/python-modules/python-fluent/fluent-syntax.nix

```
@@ -0,0 +1,42 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  setuptools,
+  typing-extensions,
+}:
+
+let
+  version = "0.19.0";
+
+  src = fetchFromGitHub {
+    owner = "projectfluent";
+    repo = "python-fluent";
+    rev = "fluent.syntax@${version}";
+    hash = "sha256-nULngwBG/ebICRDi6HMHBdT+r/oq6tbDL7C1iMZpMsA=";
+  };
+in
+buildPythonPackage {
+  pname = "fluent-syntax";
+  inherit version;
+  pyproject = true;
+
+  inherit src;
+  sourceRoot = "${src.name}/fluent.syntax";
+
+  dependencies = [
+    setuptools
+    typing-extensions
+  ];
```

```suggestion
  build-system = [ setuptools ];

  dependencies = [ typing-extensions ];
```
`setuptools` should be a build-time dependency.
https://github.com/projectfluent/python-fluent/blob/6836ef49c0fab588cd842b1bb310e43a91df0b34/fluent.syntax/setup.py#L29C7-L31


Could you please add `build-system` to others as well?